src/settings_internal.h

```diff
@@ -164,6 +164,12 @@ struct IntSettingDesc : SettingDesc {
 	 * @param The new value for the setting.
 	 */
 	typedef void PostChangeCallback(int32_t value);
+	/**
+	 * A callback to set the correct default value, in case it can be measured in time
+	 * units or expressed as a percentage.
```

Comment:
This  seems to exclude other use cases given the wording. Might you have meant this as an example?
```suggestion
	 * A callback to set the correct default value. For example a default that can be measured in time
	 * units or expressed as a percentage.
```

Reply:
Yes, looks better. Since I've have change the callback to return the correct value, I will also replace "set" with "get".